packages/mdc-icon-button/_mixins.scss

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+@import "@material/theme/mixins";
+@import "./variables";
+
+@mixin mdc-icon-button-size($width, $height: $width, $padding: max($width, $height)/2) {
+  width: $width + $padding * 2;
+  height: $height + $padding * 2;
+  padding: $padding;
+
+  // stylelint-disable-next-line selector-max-type
+  svg,
+  img {
+    width: $width;
+    height: $height;
+  }
+}
+
+@mixin mdc-icon-button-font-size($font-size) {
+  @include mdc-icon-button-size($font-size);
+
+  font-size: $font-size;
+}
+
+@mixin mdc-icon-button-ink-color($color) {
+  @include mdc-theme-prop(color, $color);
+  @include mdc-states($color);
+}
+
+@mixin mdc-icon-button-base_() {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius-unbounded;
+  @include mdc-icon-button-font-size(24px);
+
+  display: inline-block;
+  position: relative;
+  align-items: center;
```

Comment:
Remove `align-items` and `justify-content` since they no longer apply now that display is inline-block